Public/Src/IDE/Generator/CommonBuildFiles/CSharp.targets

```diff
@@ -7,6 +7,5 @@
     <LangVersion Condition="'$(LangVersion)' == 'preview' AND '$(VisualStudioVersion)' == '15.0'">latest</LangVersion>
   </PropertyGroup>
 
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />

```

Comment:
>$(MSBuildToolsPath)\Microsoft.CSharp.targets [](start = 19, length = 44)

Probably a datapoint from your survey but I'm skeptical unit tests will work as expected without our funny deployment and test execution logic. Not that the VS test integration we had previously was very dialed in anyway.

Reply:
Agreed.  I didn't focus much on tests, but will do in a next iteration